Cargo.toml

```diff
@@ -22,6 +22,7 @@ tracing = "0.1.41"
 tracing-subscriber = { version = "0.3.19", features = ["env-filter", "json"] }
 ureq = { version = "2.12.1" }
 reliquary = { git = "https://github.com/IceDynamix/reliquary", tag = "v6.2.0" }
+self_update = "0.41.0"
```

Comment:
Since self-updating is only implemented for Windows, the dependency should also only be required for Windows.

Reply:
Good catch, I meant to do this.